cypress/plugins/index.js

```diff
@@ -1,17 +1,68 @@
-// ***********************************************************
-// This example plugins/index.js can be used to load plugins
-//
-// You can change the location of this file or turn off loading
-// the plugins file with the 'pluginsFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/plugins-guide
-// ***********************************************************
-
-// This function is called when a project is opened or re-opened (e.g. due to
-// the project's config changing)
-/* eslint-disable no-unused-vars */
-module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
+const fs = require('fs');
+const path = require('path');
+const env = require('../../config/env.json');
+
+module.exports = on => {
+  on('task', {
+    // return an array of all challenge paths
+    getAllChallengePaths(targetBlock) {
```

Comment:
It should be easier to make use of `tools/scripts/build/build-curriculum.js` or `getChallengesForLang`, that it uses, to get the data you need here.

Reason being, that's how we construct the curriculum, the Cypress testing should automatically update as and when we change that.  It is, slightly costly in terms of time, but it's only a few seconds.

Reply:
Did not know this existed... I'll take a look at it.

Reply:
It's pretty new, we needed it so that the api could be created without requiring the whole of `/curriculum`.

Reply:
I am calling `getChallengesForLang` but I can't seem to find the keys in the object.  calling obj['responsive-web-design'].